src/views/pam/RiskDetect/AccountCheckDetail/index.vue

```diff
@@ -8,25 +8,24 @@
 
 <script>
 import { GenericDetailPage } from '@/layout/components'
-import AccountPushInfo from './AccountPushInfo'
+import AccountCheckDetail from './Detail.vue'
 
 export default {
   components: {
     GenericDetailPage,
-    AccountPushInfo
+    AccountCheckDetail
   },
   data() {
     return {
       plan: { name: '', username: '', comment: '' },
       config: {
         titlePrefix: this.$t('AccountCheckDetail'),
-        activeMenu: 'AccountPushInfo',
-        url: '/api/v1/accounts/push-account-automations',
+        url: '/api/v1/accounts/check-account-automations',
         submenu: [
           {
             title: this.$t('Basic'),
-            name: 'AccountPushInfo',
-            hidden: () => !this.$hasPerm('accounts.view_changesecretautomation')
+            name: 'AccountCheckDetail',
+            hidden: () => !this.$hasPerm('accounts.view_accountrisk')
           }
         ]
       }
```

Comment:
The above code is already formatted and appears to be correct, so there are no obvious issues or differences worth discussing.

Since my last update was in late 2021, I would recommend reviewing the current version of this code for potential modifications that may have been introduced since then.